react/features/prejoin/components/Prejoin.js

```diff
@@ -308,6 +309,14 @@ class Prejoin extends Component<Props, State> {
                                     type = 'secondary'>
                                     { 'Login with Superhero' }
                                 </ActionButton>}
+
+                                <div>
+                                    <TipButton
+                                        account = 'ak_21vaVFzQ98f4Kf9RoBSZDSr9zaiHfrni8PPDYMamuandeSogMA'
```

Comment:
Are we hard coding an AK address?

Reply:
it's for the test. and I open PR just to see the diff

Reply:
Ok. got it.